salt/journal-cms/scripts/restore-legacy.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# restore legacy files. run as root
+set -exv
+
+# download the legacy files
+cd /opt/ubr/
+./ubr.sh download s3 adhoc _platform.sh/legacy_cms.sql.gz _platform.sh/legacy_cms_files.tar.gz
+
+# the database can be handled with ubr
+./ubr.sh restore file adhoc /tmp/ubr/legacy_cms.sql.gz mysql-database.legacy_cms
```

Comment:
> If so ok, but where did it get the configuration about the host, username and 
> password to do so?

ubr has a cfg file with the machine's permissions, rds host, port etc. auth details  previously lived in `/etc/ubr/config` and was `source`ed as env vars, but I decided that was shit and that explicit was better (passing variables through envvars appears to be a fad I was passing through with AWS beanstalk and what the Ruby guys prefer. I know better now).